### PR TITLE
Create PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by pilhuhn**

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+** Describe the change **
+
+_explantation of what it does_
+
+** Issue reference **
+
+* If this is a fix for a JIRA, please put JIRA-reference in the PR title like _[KIALI-7] New icons_
```
Comment (xeviknal) [inline PULL_REQUEST_TEMPLATE.md]: After voting for PR titles, we decided to put JIRA reference without the square brackets.

Comment (pilhuhn) [reply PULL_REQUEST_TEMPLATE.md]: Ok, good catch.

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+** Describe the change **
+
+_explantation of what it does_
```
Comment (abonas) [inline PULL_REQUEST_TEMPLATE.md]: explanation

Comment (abonas) [reply PULL_REQUEST_TEMPLATE.md]: and I think "Please describe what is the code change about" is more clear

File: PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+** Issue reference **
+
+* If this is a fix for a JIRA, please put JIRA-reference in the PR title like _`KIALI-7 New icons`_ (Jira-id, one space, description)
+* If this is a fix for a GH-issue, please link it here
+
+** Backwards in compatible? **
```
Comment (abonas) [inline PULL_REQUEST_TEMPLATE.md]: afaik incompatible is one word

File: PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+[ ] Is your change introducing backwards incompatible changes?
+
+_describe the incomabibilities if appropriate_
```
Comment (abonas) [inline PULL_REQUEST_TEMPLATE.md]: incomabibilities is definitely not a word :)
I propose "please describe in what way the change is incompatible and how does it influence Kiali and its users"

File: PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+_explantation of what it does_
+
+** Issue reference **
+
+* If this is a fix for a JIRA, please put JIRA-reference in the PR title like _`KIALI-7 New icons`_ (Jira-id, one space, description)
+* If this is a fix for a GH-issue, please link it here
+
+** Backwards in compatible? **
+
+[ ] Is your change introducing backwards incompatible changes?
```
Comment (abonas) [inline PULL_REQUEST_TEMPLATE.md]: I think that line 10 and 12 are practically the same, I would leave only one of them

File: PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+** Issue reference **
+
+* If this is a fix for a JIRA, please put JIRA-reference in the PR title like _`KIALI-7 New icons`_ (Jira-id, one space, description)
+* If this is a fix for a GH-issue, please link it here
+
+** Backwards incompatible? **
+
+[ ] Is your change introducing backwards incompatible changes?
+
+_describe the incomabibilities if appropriate_
```
Comment (abonas) [inline PULL_REQUEST_TEMPLATE.md]: incomabibilities is not a word afaik :)

**Comment by abonas** [comment]
@pilhuhn for some reason I do not have an option to squash, only regular merge. I approved, go ahead squash merge.

**Pull request merged**